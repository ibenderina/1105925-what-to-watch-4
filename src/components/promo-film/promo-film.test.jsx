import PromoFilm from "@components/promo-film/promo-film";

it(`Should PromoFilm render correctly`, () => {
  const commonText = `Hello`;
  let tree;

  window.act(() => {
    tree = window.create(<PromoFilm
      film={
        {
          title: commonText,
          src: commonText,
          genre: commonText,
          date: commonText,
          background: commonText,
        }
      }
    />);
  });
  expect(tree.toJSON()).toMatchSnapshot();
});
