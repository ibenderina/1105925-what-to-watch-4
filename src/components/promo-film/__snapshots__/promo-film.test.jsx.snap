// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PromoFilm render correctly 1`] = `
<section
  className="movie-card"
>
  <div
    className="movie-card__bg"
  >
    <img
      alt="Hi background"
      src="Hi"
    />
  </div>
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>
  <header
    className="page-header movie-card__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="user-block"
    >
      <div
        className="user-block__avatar"
      >
        <img
          alt="User avatar"
          height="63"
          src="img/avatar.jpg"
          width="63"
        />
      </div>
    </div>
  </header>
  <div
    className="movie-card__wrap"
  >
    <div
      className="movie-card__info"
    >
      <div
        className="movie-card__poster"
      >
        <img
          alt="Hi poster"
          height="327"
          src="Hi"
          width="218"
        />
      </div>
      <div
        className="movie-card__desc"
      >
        <h2
          className="movie-card__title"
        >
          Hi
        </h2>
        <p
          className="movie-card__meta"
        >
          <span
            className="movie-card__genre"
          >
            Hi
          </span>
          <span
            className="movie-card__year"
          >
            Hi
          </span>
        </p>
        <div
          className="movie-card__buttons"
        >
          <button
            className="btn btn--play movie-card__button"
            type="button"
          >
            <svg
              height="19"
              viewBox="0 0 19 19"
              width="19"
            >
              <use
                href="#play-s"
              />
            </svg>
            <span>
              Play
            </span>
          </button>
          <button
            className="btn btn--list movie-card__button"
            type="button"
          >
            <svg
              height="20"
              viewBox="0 0 19 20"
              width="19"
            >
              <use
                href="#add"
              />
            </svg>
            <span>
              My list
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</section>
`;
